src/__tests__/spectral.jest.test.ts

```diff
@@ -110,15 +110,15 @@ describe('Spectral', () => {
       expect.arrayContaining([
         expect.objectContaining({
           code: 'oas2-schema',
-          path: ['paths', '/todos/{todoId}', 'get', 'responses', '200'],
+          path: ['paths', '/todos/{todoId}', 'get', 'responses', '200', 'description'],
           range: {
             end: {
-              character: 11,
-              line: 31,
+              character: 29,
+              line: 17,
             },
             start: {
-              character: 17,
-              line: 16,
+              character: 27,
+              line: 17,
```

Comment:
This one is a bit crazy.  Spectral reports
```
 17:17    error  oas2-schema                  Property `description` is not expected to be here.
```
on the "responses/200" object even before my changes.  No idea why -- `description` is not only expected, it is _required_ in response objects.

So the changes here are simply making this more precise, pointing to the specific property with more precise range information.